cpp/betweenness_centrality_module/betweenness_centrality_module.cpp

```diff
@@ -53,12 +55,15 @@ extern "C" int mgp_init_module(mgp_module *module, mgp_memory *memory) {
   // Query module arguments
   auto bool_value_directed = mgp_value_make_bool(true, memory);
   auto bool_value_normalized = mgp_value_make_bool(true, memory);
+  auto int_value_threads = mgp_value_make_int(8, memory);
```

Comment:
Instead of hardcoding 8, use this https://en.cppreference.com/w/cpp/thread/thread/hardware_concurrency